src/script/lua_api/l_mainmenu.cpp

```diff
@@ -540,8 +540,12 @@ int ModApiMainMenu::l_create_world(lua_State *L)
 			+ sanitizeDirName(name, "world_");
 
 	std::vector<SubgameSpec> games = getAvailableGames();
-	if (gameidx < 0 || gameidx >= (int) games.size()) {
-		lua_pushstring(L, "Invalid game index");
+	auto game_it = std::find_if(games.begin(), games.end(), [gameid] (const SubgameSpec &spec)
+	{
```

Comment:
not sure if we have a standard for this but IMO this is less weird
```suggestion
	auto game_it = std::find_if(games.begin(), games.end(), [gameid] (const SubgameSpec &spec) {
```